docs/image.md

```diff
@@ -5,7 +5,8 @@ kpack will monitor the inputs to the image configuration to rebuild the image wh
 
 The following defines the relevant fields of the `image` resource spec in more detail:
 
-- `tag`: The image tag.
+- `tag`: The image tag. This is the primary image tag that is immutable for a given image and is re-used during subsequent builds for certain cached data.
+- `additionalTags`: Any additional list of image tags that should be published. This list of tags is mutable.
```

Comment:
I am a bit conflicted if we should only allow additional tags to be tag names or fully fledged docker image tags. The former would be easier in terms of secret management etc. and ensuring that pushes succeed where as the latter would be more flexible.  I am tempted to switch to the former but I have currently implemented the latter.

Reply:
This seems relevant: https://github.com/buildpacks/community/discussions/128